### DISCUSSION TITLE
MQTT Dynamic DNS

### DISCUSSION THREAD

**Discussion by cucumberfail** [tbnobody/OpenDTU#966] [Ideas]
Hi everyone,

my iobroker is on a home server and the OpenDTU at my Parents House,
i would love to use it without typing in my IP every few days after it gets renewed by my isp and stopped working.
would it be possible to add DynamicDNS hostnames to work with OpenDTU, don't get me wrong i already have one that is working with everything else *.synology.me but it doesn't work with OpenDTU.

![dynamicip](https://github.com/tbnobody/OpenDTU/assets/134771678/f089325f-d293-4c77-9d8e-cc4938aacbce)

i can only add my direct ip into that field for it to work over internet.
thank you

Yours sincerely,
cucumberfail

**Comment by tbnobody** [comment]
DNS name resolution works perfectly with OpenDTU as it does not implement it anything by itself but just uses the existing ESP32 infrastructure.
Please have a look at Info --> Network and doublecheck if your DNS servers IP is set corretly.

**Comment by cucumberfail** [comment]
Thank you, i disabled dhcp and set my routers ip as dns in OpenDTU network settings.
normally all dhcp clients get the piholes dns for adblocking and the routers ip as gateway.
i took that out of equation and now its working fine.
